arelle/ModelXbrl.py

```diff
@@ -108,6 +111,7 @@ def create(modelManager, newDocumentType=None, url=None, schemaRefs=None, create
                 loadSchemalocatedSchemas(modelXbrl)
     return modelXbrl
 
+
 def loadSchemalocatedSchemas(modelXbrl) -> None:
     from arelle import ModelDocument
```

Comment:
```suggestion
    from arelle.ModelDocument import ModelDocument
```
This is not a change from this PR, but it affects lines 119 and 122